lib/core_extensions/relation/human_count.rb

```diff
@@ -7,7 +7,7 @@ module HumanCount
       # > user.last.searches.human_count
       # => "12 recherches"
       def human_count
-        "#{count} #{@klass.model_name.human(count: count).downcase}"
+        "#{size} #{@klass.model_name.human(count: size).downcase}"
```

Comment:
Ca change quoi de passer de `count` à `size`  ?

Reply:
http://web.archive.org/web/20100210204319/http://blog.hasmanythrough.com/2008/2/27/count-length-size

> In ActiveRecord, there are several ways to find out how many records are in an association, and there are some subtle differences in how they work.
> 
> * post.comments.count - Determine the number of elements with an SQL COUNT query. You can also specify conditions to count only a subset of the associated elements (e.g. :conditions => {:author_name => "josh"}). If you set up a counter cache on the association, #count will return that cached value instead of executing a new query.
> * post.comments.length - This always loads the contents of the association into memory, then returns the number of elements loaded. Note that this won't force an update if the association had been previously loaded and then new comments were created through another way (e.g. Comment.create(...) instead of post.comments.create(...)).
> * post.comments.size - This works as a combination of the two previous options. If the collection has already been loaded, it will return its length just like calling #length. If it hasn't been loaded yet, it's like calling #count.


En fait, ruby utilise plutôt .length ou .size; c’est mon passé en objective-c qui me fait écrire .count sans réfléchir.